examples/tidyBench.ts

```diff
@@ -0,0 +1,38 @@
+import * as sm from '@shumai/shumai'
+import { noInline, startSamplingProfiler } from 'bun:jsc'
+
+startSamplingProfiler(Bun.main + '.prof', 10)
+
+const r = (v) => {
+  return Math.round(v * 1e3) / 1e3
+}
+
+function tidy_mm_pw(size: number, iters: number, warmup: number) {
+  const N = size
+  const a = sm.randn([N, 64])
+  const b = sm.identity(N)
+  const c = sm.randn([N, 1])
+
+  const op = (K) => {
+    let d = a
+    for (let i = 0; i < K; ++i) {
+      d = b.matmul(d)
+      for (let j = 0; j < 5; ++j) {
+        d = d.add(c)
+      }
+    }
+    const o = d.sum().toFloat32()
+    return o
+  }
+
+  sm.util.tidy(op, warmup)
+  const t0 = performance.now()
+  sm.util.tidy(op, [iters])
```

Comment:
so you'll actually want to run `tidy` within a loop of repeated iterations.  Right now I think this isn't actually doing much compared to `messy` - just collecting and freeing a lot of Tensors at the end.

Reply:
Ahh, yea that would be why `tidy` was so expensive in the benchmark haha; I should probably debug/profile before 2 AM when I'm not as likely to miss that haha.

I'm not intending to add the benchmark to the codebase (actually deleted as it was accidentally committed), so resolved on this.